chia/cmds/coins.py

```diff
@@ -25,30 +25,26 @@ def coins_cmd(ctx: click.Context) -> None:
 )
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which wallet to use", type=int)
 @click.option("-i", "--id", help="Id of the wallet to use", type=int, default=1, show_default=True, required=True)
-@click.option("-u", "--show_unconfirmed", help="Separately display unconfirmed coins.", is_flag=True)
+@click.option("-u", "--show-unconfirmed", help="Separately display unconfirmed coins.", is_flag=True)
 @click.option(
-    "-a",
-    "--min-coin-amount",
+    "--min-amount",
     help="Ignore coins worth less then this much XCH or CAT units",
     type=str,
     default="0",
 )
 @click.option(
-    "-l",
-    "--max-coin-amount",
+    "--max-amount",
     help="Ignore coins worth more then this much XCH or CAT units",
     type=str,
     default="0",
 )
 @click.option(
-    "-e",
-    "--excluded-coin-ids",
+    "--exclude-id",
```

Comment:
Make the parameter/variable name also sensible
```suggestion
    "--exclude-id",
    "excluded_ids",
```